pages/api/v1/assets/[asset_hash]/index.js

```diff
@@ -26,7 +26,8 @@ export default async function handle(req, res) {
 			return res.status(404).json({ message: 'Asset does not exist on our records' })
 		}
 
-		const filePath = path.join(process.cwd(), "nftdata", asset.fileName)
+		const STORAGE_DIR = process.env.STORAGE_DIR
+		const filePath = path.join(STORAGE_DIR, asset.fileName)
```

Comment:
Adding to the storage path comment above it might be unexpected that `asset.fileName` is in fact not a file name but a path relative to `STORAGE_DIR`.

Reply:
There is a "good" reason for that; 
docker-compose uses the build-stage directly for development (to build on the fly), while production uses the runner. 

Will follow-up on this in #19